sw/applications/example_ams_peripheral/main.c

```diff
@@ -19,29 +19,22 @@ int main(int argc, char *argv[])
         unsigned int AMS_START_ADDRESS = EXT_PERIPHERAL_START_ADDRESS + 0x1000;
         mmio_region_t ams_base_addr = mmio_region_from_addr((uintptr_t)AMS_START_ADDRESS);
 
-        printf("0");
```

Comment:
pls keep these printf in this specific example

Reply:
compulsory (it is an analog example, we need those...)

Reply:
done